### PR TITLE
Replace Entity with EntityInterface in CrudViewHelper

### PR DESCRIPTION
**Pull request by Asgraf**
\Cake\ORM\Entity requirement should be replaced with \Cake\Datasource\EntityInterface
This would make CrudViewHelper play nicely with other Entity classes like Muffin\Webservice\Model\Resource

### CODE REVIEW

File: src/View/Helper/CrudViewHelper.php [hunk 1]
```diff
@@ -66,11 +66,11 @@ public function getContext()
      * Process a single field into an output
      *
      * @param string $field The field to process.
-     * @param Entity $data The entity data.
+     * @param EntityInterface $data The entity data.
```
Comment (ADmad) [inline src/View/Helper/CrudViewHelper.php]: Should be FQCN

File: src/View/Helper/CrudViewHelper.php [hunk 2]
```diff
@@ -102,11 +102,11 @@ public function process($field, Entity $data, array $options = [])
     /**
      * Get the current field value
      *
-     * @param Entity $data The entity data.
+     * @param EntityInterface $data The entity data.
```
Comment (ADmad) [inline src/View/Helper/CrudViewHelper.php]: Should be FQCN

**Comment by Asgraf** [comment]
@ADmad a have added FQCNs as you requested

**Comment by Asgraf** [comment]
Is this patch will be merged?

**Comment by ADmad** [comment]
Sorry, we forgot about this one. I'll have a look again and merge.

**Pull request merged**